packages/frontend/src/components/DashboardTiles/TileBase/TileBase.module.css

```diff
@@ -0,0 +1,32 @@
+.tileWrapper {
+    position: relative;
+    height: 100%;
+
+    &:hover .drag-handle-icon {
+        opacity: 1;
+    }
+}
+
+.tileCard {
+    overflow: unset;
+    border: 1px solid
+        light-dark(
+            var(--mantine-color-ldGray-1),
+            color-mix(in srgb, var(--mantine-color-ldDark-1), white 5%)
+        );
```

Comment:
new border colors are quite strong compared to prev. :thinking_face:

_Before_

![CleanShot 2025-12-23 at 17.47.22.png](https://app.graphite.com/user-attachments/assets/bc795b86-d349-46dd-9873-3c92aff9ac4b.png)

_This PR_

![CleanShot 2025-12-23 at 17.46.49.png](https://app.graphite.com/user-attachments/assets/31c76cc2-9725-4233-b85f-c78164e101f3.png)

I think we will work on this in the tile refactor?  #17997

Reply:
good catch - fixed!